src/rgw/rgw_lc.cc

```diff
@@ -302,12 +302,12 @@ static inline std::ostream& operator<<(std::ostream &os, rgw::sal::Lifecycle::LC
     return os;
 }
 
-int RGWLC::bucket_lc_prepare(int index, LCWorker* worker)
+int RGWLC::bucket_lc_prepare(const DoutPrefixProvider *dpp, int index, LCWorker* worker)
 {
   vector<rgw::sal::Lifecycle::LCEntry> entries;
   string marker;
 
-  dout(5) << "RGWLC::bucket_lc_prepare(): PREPARE "
+  ldpp_dout(dpp, 5) << "RGWLC::bucket_lc_prepare(): PREPARE "
```

Comment:
The dpp here is not needed, it should be a `this` since `RGWLC` is a `DoutPrefixProvider`.